x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -327,11 +326,7 @@ export const RulesList = ({
       try {
         const result = await loadActionTypes({ http });
         const sortedResult = result
-          .filter(
-            // TODO: Remove "DEFAULT_HIDDEN_ACTION_TYPES" when cases connector is available across Kibana.
-            // Issue: https://github.com/elastic/kibana/issues/82502.
-            ({ id }) => actionTypeRegistry.has(id) && !DEFAULT_HIDDEN_ACTION_TYPES.includes(id)
-          )
+          .filter(({ id }) => actionTypeRegistry.has(id))
```

Comment:
This is the drop down list that shows up on the Rule List view. Not sure if we should filter this list to only connectors registered for the `alerting` feature?

<img width="2205" alt="Screen Shot 2022-07-18 at 5 49 14 PM" src="https://user-images.githubusercontent.com/13104637/179623385-c5bd8ac5-f562-46d5-9889-2d596dce8f7b.png">

cc @mikecote

Reply:
Good catch, I think that would make sense 👍